build.gradle.kts

```diff
@@ -47,7 +47,7 @@ val projectProps by tasks.registering(WriteProperties::class) {
     description = "Write project properties in a file."
 
     // Set output file to build/project.properties
-    outputFile = file("$buildDir/codyze.properties")
+    destinationFile = file("${layout.buildDirectory.get()}/codyze.properties")
```

Comment:
I think in Kotlin the explicit .get() call could be omitted.

Reply:
Without `.get()` the property file ends up in a weird folder like `property 'buildDir'`. It's caused by `buildDirectory`, which is a `DirectoryProperty`, and our string concatenation. I've looked through the API and found an alternative (i.e., the intended approach).